forge-gui/res/cardsfolder/upcoming/marshal_of_zhalfir.txt

```diff
@@ -0,0 +1,9 @@
+Name:Marshal of Zhalfir
+ManaCost:W U
+Types:Creature Human Knight
+PT:2/2
+S:Mode$ Continuous | Affected$ Creature.Knight+Other+YouCtrl | AddPower$ 1 | AddToughness$ 1 | Description$ Other Knights you control get +1/+1.
+A:AB$ Tap | Cost$ W U T | ValidTgts$ Creature.Other | TgtPrompt$ Select another target creature | SpellDescription$ Tap another target creature.
+SVar:PlayMain1:TRUE
+DeckNeeds:Type$Knight
```

Comment:
I'm not 100% sure this is a Needs because of the other line of text... definitely Hints
Your call!